### PR TITLE
frontend home page

### PR DESCRIPTION
**Pull request by kyudori**
# Description

LPVS Front-End Home Page.
User, History, and Dashboard pages will be uploaded.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code cleanup/refactoring
- [ ] Documentation update
- [ ] This change requires a documentation update
- [ ] CI system update
- [ ] Test Coverage update

# How Has This Been Tested?
$npm install
$npm start
Go to http://localhost:3000 with Chrome.
image

**Test Configuration**:
* Java: v11
* LPVS Release: v1.1.0

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by o-kopysov — APPROVED** [review]
LGTM

File: frontend/.env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+REACT_APP_GOOGLE_REST_API_KEY={type_your_key}
+REACT_APP_NAVER_REST_API_KEY={type_your_key}
+REACT_APP_KAKAO_REST_API_KEY={type_your_key}
```
Comment (m-rudyk) [inline frontend/.env:3]: It is better to add empty line at the end of file here and few other places in this PR.

**Comment by o-kopysov** [comment]
Also I recommend to add file headers into the files with source code.

**Comment by tiokim** [comment]
> Also I recommend to add file headers into the files with source code.

Do you mean copyright/license header?

**Comment by o-kopysov** [comment]
> Do you mean copyright/license header?

Yes, exactly.

**Review by o-kopysov — APPROVED** [review]
LGTM

File: frontend/public/index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2023 kyudori, Basaeng, hwan5180, quswjdgma83
+ *
+ * Use of this source code is governed by a MIT license that can be
+ * found in the LICENSE file.
+ */
+
```
Comment (tiokim) [inline frontend/public/index.html:7]: It should be commented out.

**Review by tiokim — CHANGES_REQUESTED** [review]

File: frontend/public/index.html (hunk 2 above)
Comment (kyudori) [reply frontend/public/index.html]: Done.

**Review by tiokim — APPROVED** [review]
LGTM

**Pull request merged**